compass/ocean/mesh/lts_regions.py

```diff
@@ -0,0 +1,310 @@
+import math
+import subprocess as sp
+
+import netCDF4 as nc
+import numpy as np
+import xarray
+from shapely.geometry import Point
+from shapely.geometry.polygon import Polygon
+
+from compass.step import Step
+
+
+class LTSRegionsStep(Step):
+    """
+    A step for adding LTS regions to a global MPAS-Ocean mesh
+
+    Attributes
+    ----------
+    cull_mesh_step : CullMeshStep
+        The culled mesh step containing input files to this step
```

Comment:
This needs to be the full path so it shows up right in the auto-generated documentation.
```suggestion
    cull_mesh_step : compass.ocean.mesh.cull.CullMeshStep
        The culled mesh step containing input files to this step
```